pkg/storegateway/parquet_bucket_stores.go

```diff
@@ -125,24 +136,24 @@ func NewParquetBucketStores(
 
 	// Register metrics.
 	stores.syncTimes = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
-		Name:    "cortex_bucket_stores_blocks_sync_seconds",
+		Name:    "cortex_parquet_bucket_stores_blocks_sync_seconds",
 		Help:    "The total time it takes to perform a sync stores",
 		Buckets: []float64{0.1, 1, 10, 30, 60, 120, 300, 600, 900},
 	})
 	stores.syncLastSuccess = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-		Name: "cortex_bucket_stores_blocks_last_successful_sync_timestamp_seconds",
+		Name: "cortex_parquet_bucket_stores_blocks_last_successful_sync_timestamp_seconds",
 		Help: "Unix timestamp of the last successful blocks sync.",
 	})
 	stores.tenantsDiscovered = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-		Name: "cortex_bucket_stores_tenants_discovered",
+		Name: "cortex_parquet_bucket_stores_tenants_discovered",
 		Help: "Number of tenants discovered in the bucket.",
 	})
 	stores.tenantsSynced = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-		Name: "cortex_bucket_stores_tenants_synced",
+		Name: "cortex_parquet_bucket_stores_tenants_synced",
 		Help: "Number of tenants synced.",
 	})
 	stores.blocksLoaded = prometheus.NewDesc(
-		"cortex_bucket_store_blocks_loaded",
+		"cortex_parquet_bucket_store_blocks_loaded",
```

Comment:
Renaming these metrics is just a nit i commented some time ago in some other PR.